charts/rancher-turtles/templates/core-provider.yaml

```
@@ -0,0 +1,71 @@
+{{- if .Values.clusterAPI.enabled }}
```

I believe you’ve hit the issue with the naming here. I looked into this in the uninstall [PR](https://github.com/kubernetes-sigs/cluster-api-operator/pull/251/files#diff-5f2ca550f95c3e34df471651fd2d2a0ce91c55c35127672c7ed6ebbdd0109613R1) You can still use the 
`cluster-api-operator` name like this:
```suggestion
{{- if index .Values "cluster-api-operator" "enabled" }}
```